opensearch-maps-server/Dockerfile

```diff
@@ -1,7 +1,11 @@
-FROM node:16
+FROM amazonlinux:2
 
 WORKDIR /usr/src/app
 
+RUN curl -sL https://rpm.nodesource.com/setup_16.x | bash - && \
+    yum install -y nodejs wget tar gzip && \
```

Comment:
[just a heads-up] install only those packages that are required in your docker image. Adding additional packages leads to burden of maintaining them in the future : )

Reply:
Thanks! Yes, these packages are needed to run the docker image.